concourse/Makefile

```
@@ -15,7 +15,7 @@ CERTIFICATION_PIPELINE_NAME ?= pxf-certification
 CLOUDBUILD_PIPELINE_NAME    ?= cloudbuild
 DEV_CLOUDBUILD_PIPELINE_NAME ?= dev-cloudbuild-$(USER)-$(BRANCH)
 PERF_PIPELINE_NAME          ?= pxf_perf-$(SCALE)g-RHEL$(REDHAT_MAJOR_VERSION)
-PIVNET_PIPELINE_NAME        ?= pivnet_artifacts
+GCS_PIPELINE_NAME           ?= gpdb_gcs_artifacts
```

since we're changing the name anyway, let's switch to dashs 
```suggestion
GCS_PIPELINE_NAME           ?= gpdb-gcs-artifacts
```